Automattic/WP-Job-Manager
# Update notices

Fixes #2581

Based on #2614 for shared styles

### Changes Proposed in this Pull Request

* Update notice styles
* Bring all notices to use the new notice system
* Use a single 'license required' notice for all add-on license issues
* Limit showing most of the add-ons to main job listing page and settings
* Various copy & action improvements
* Add a few helper methods to get the product icon for an add-on, and a few fixes to the license keys page

### Testing Instructions

* Playground: [Existing site](https://wpjobmanager.com/playground/?core=2023/10/2620-7e20a6fb&url=/wp-admin) | [Fresh site](https://wpjobmanager.com/playground/?core=2023/10/2620-7e20a6fb&site=empty&url=/wp-admin)

* Reset dismissed notices: `wp option delete wp_job_manager_dismissed_notices`
* Reset usage tracking notice:
  * Turn off usage tracking
  * `wp option delete job_manager_usage_tracking_opt_in_hide`
  * `wp option delete job_manager_display_usage_tracking_once`
* Updates, licenses:
  * Add older versions of a few add-ons from https://github.com/Automattic/wpjm-addons/releases
  * In `wp shell`:
  * `include_once JOB_MANAGER_PLUGIN_DIR . '/includes/helper/class-wp-job-manager-helper.php'; WP_Job_Manager_Helper::instance()->check_for_updates([]);`


### Release Notes

* Clean up, update and style admin notices 

### New or Updated Hooks and Templates


*


### Deprecated Code

*

### Screenshot / Video

<img width="1312" alt="image" src="https://github.com/Automattic/WP-Job-Manager/assets/176949/46a4972d-2308-4e35-9575-4a7c89825ab6">



----

| Plugin build for 4cdde5d19813464482a892177016d299c7909923 <a href="#"><img width=600></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/10/wp-job-manager-zip-2620-4cdde5d1.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/10/2620-4cdde5d1)             |
